src/IBM.Cloud.SDK.Core/Util/CredentialUtils.cs

```diff
@@ -132,37 +132,37 @@ private static List<string> GetFilesToCheck()
             {
                 files.Add(userSpecifedPath);
             }
-
-            if (!string.IsNullOrEmpty(unixHomeDirectory))
+            
+            if (!string.IsNullOrEmpty(projectDirectory))

```

Comment:
I'm curious why we call it "projectDirectory"...  I think technically it's the "current working directory", right?

Reply:
hmm yeah thats the current working directory - for .net its different depending on the build target. Not sure what a better name could be - `projectBuildDirectory`?

Reply:
why not just "currentDirectory" or "cwd"?   I mean... what does "project" even mean in this context?   We're talking about the current working directory that might be in effect when the application is running, right?

Reply:
OK I changed it to currentDirectory - which makes sense from the built project's perspective.